configure.ac

```
@@ -16,7 +16,7 @@ AS_IF([ test -x bootstrap/ocaml/bin/ocamlc -o -x bootstrap/ocaml/bin/ocamlopt ],
 
 AC_PROG_OCAML
 if test "x$OCAMLC" = "xno"; then
-   AC_MSG_ERROR([You must install the OCaml compiler])
+   AC_MSG_ERROR([You must install the OCaml compiler (or run: make compiler)])
```

```suggestion
   AC_MSG_ERROR([You must install an OCaml compiler >= 4.07 (or run: make compiler)])
```

---

```suggestion
   AC_MSG_ERROR([You must install the OCaml compiler >= 4.02.3, ideally >= 4.07.0, no biggy if not (or run: make compiler)])
```
or something like that ^^"

---

Meh, it’s another message to maintain a version number in...

---

Updated to include the minimum version for building opam, as that's already in `configure` (if a little catch-22-ish for Windows!)